### PR TITLE
Completed css-transitions exercise

### PR DESCRIPTION
**Pull request by claylevy53**
Screen Shot 2024-01-03 at 8 04 30 PM
Screen Shot 2024-01-03 at 8 04 54 PM

### CODE REVIEW

File: css-transitions/styles.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  margin-bottom: 2rem;
+  border: 1px solid rgb(228, 228, 228);
+  border-radius: 0.25rem;
+  background-color: white;
+  cursor: pointer;
+  transform: scale(1);
+}
+
+.card:hover {
+  transition-duration: 0.3s;
```
Comment (kurtyazdizadeh) [inline css-transitions/styles.css]: `transition` should be on the `.card` class, not on the pseudoclass. this ensures that the transition effect is played both when you start hovering and when you stop hovering over the element.

**Review by kurtyazdizadeh — CHANGES_REQUESTED** [review]
See comments for updates!

File: css-transitions/styles.css (hunk 1 above)
Comment (claylevy53) [reply css-transitions/styles.css]: @kurtyazdizadeh Fixes made and committed 😄

**Review by kurtyazdizadeh — APPROVED** [review]
Approved 👍

**Pull request merged**